src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -86,9 +96,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setCompany(companyToEdit, editedCompany);
         model.updateFilteredCompanyList(PREDICATE_SHOW_ALL_COMPANIES);
-
         model.setCurrentViewedCompany(editedCompany);
-
         return new CommandResult(String.format(MESSAGE_EDIT_COMPANY_SUCCESS, Messages.format(editedCompany)));
```

Comment:
The success message can be edited to a shorter one, showing the company name only